G/GTK4/build_tarballs.jl

```diff
@@ -79,7 +144,16 @@ rm ${bindir}/gdk-pixbuf-pixdata ${bindir}/glib-compile-{resources,schemas}
 
 # These are the platforms we will build for by default, unless further
 # platforms are passed in on the command line
-platforms = filter!(p -> arch(p) != "armv6l", supported_platforms())
+platforms = supported_platforms()
+
+# Many X11 dependencies have not yet been built for armv6l
```

Comment:
I don't think that's true anymore

Reply:
https://github.com/JuliaPackaging/Yggdrasil/pull/11549